powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/service/v2/TokenService.java

```diff
@@ -84,8 +84,8 @@ public TokenCreateResponse createToken(TokenCreateRequest request, PowerAuthApiA
             response.setEncryptedData(token.getEncryptedData());
             return response;
         } catch (Exception ex) {
-            logger.warn("Creating PowerAuth token failed", ex);
-            throw new PowerAuthAuthenticationException(ex.getMessage());
+            logger.warn("Creating PowerAuth token failed, error: {}", ex.getMessage());
+            throw new PowerAuthAuthenticationException("POWER_AUTH_TOKEN_ERROR");
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.